FranticFour/Assets/01_Scripts/Controlls/AssignedController.cs

```diff
@@ -15,16 +17,17 @@ public int PlayerID
             GetControllerType();
         }
     }
-    
+
     public bool UsesMouse
     {
         get => usesMouse;
         private set => usesMouse = value;
     }
-    
 
-    [Header("Assigned controls")]
-    [SerializeField] private string vertical;
+    //Control buttons
+    [Header("Assigned controls")] [SerializeField]
```

Comment:
move [SerializeField] down one line to the same line as the variable like the rest of the script? 👍

Reply:
Fixing in next push :*